submissions/UA-girl/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+        'with longer guard hairs over a silky satin undercoat, and a long, bushy tail. ' +
+        "The breed\'s colors vary widely, with only lilac and chocolate disallowed for pedigree. Reputed for its intelligence " +
+        'and playful, gentle personality, the Maine Coon is often cited as having "dog-like" characteristics...',
+        link: 'https://en.wikipedia.org/wiki/Maine_Coon',
+        image: {
+            url: './img/maine_coon.jpg',
+            alt: 'Maine Coon cat image'
+        }
+    },
+
+    {
+        id: 3,
+        title: 'Sphynx cat',
+        text: 'The Sphynx cat is a breed of cat known for its lack of coat (fur). Hairlessness in cats is a ' +
+        'naturally occurring genetic mutation; however, the Sphynx cat, as a breed, was developed through selective breeding, ' +
+        'starting in the 1960s. The skin should have the texture of chamois leather, as it has fine hairs, ' +
+        'or the cat may be completely hairless. Whiskers may be present, either whole or broken, or may be totally absent. ' +
+        'The cats have a narrow, long head and webbed feet. Their skin is the color that their fur would be, and all ' +
+        "the usual cat markings (solid, point, van, tabby, tortie, etc.) may be found on the Sphynx cat's skin. " +
+        "Because they have no fur they lose more body heat than coated cats which makes them warm to the touch and prone to finding warm places...",
+        link: 'https://en.wikipedia.org/wiki/Sphynx_cat',
+        image: {
+            url: './img/canadian_sphinx.jpg',
+            alt: 'Canadian Sphynx cat image'
+        }
+    },
+
+    {
+        id: 4,
+        title: 'Turkish Angora',
+        text: "The Turkish Angora (Turkish: Ankara kedisi, 'Ankara cat') is a breed of domestic cat. Turkish Angoras " +
+        "are one of the ancient, natural breeds of cat, having originated in central Turkey, in the Ankara region. " +
+        "The breed has been documented as early as the 17th century and is believed to be the origin of the mutations " +
+        "for both the color white[citation needed] and long hair[citation needed]. " +
+        "The breed is also sometimes referred to as simply the Angora or Ankara cat...",
+        link: 'https://en.wikipedia.org/wiki/Turkish_Angora',
+        image: {
+            url: './img/angorra.jpg',
+            alt: 'Angorra cat image'
+        }
+    },
+];
+
+const menu = document.querySelector('.catalog-list');
+const menuItems = document.querySelectorAll('.catalog-link');
+const card = document.querySelector('.card');
+
+// menuItems.forEach(item => item.addEventListener('click', toggleCatalog));
+menu.addEventListener('click', toggleCatalog);
+menu.addEventListener('keyup', function (event) {
+    if (event.keyCode === 32) {
+        toggleCatalog(event);
+    }
+});
+
+// Animate appearance of the card
+function animate(item) {
```

Comment:
Please add info what are you going to animate.
```suggestion
function animateCard(item) {
```